include/bios_handler.hpp

```diff
@@ -60,6 +62,24 @@ class IbmBiosHandler : public BiosHandlerInterface
      * @param[in] i_msg - The callback message.
      */
     virtual void biosAttributesCallback(sdbusplus::message_t& i_msg);
+
+  private:
+    /**
+     * @brief API to read given attribute from BIOS table.
+     *
```

Comment:
could you explain below as to which table the read is done from current or pending

Reply:
The table that it is read from is internal implementation of GetAttribute API called which is not our implementation, hence agnostic to us. I can't put that here.

Whereas, the value that it returns has both current value and pending value. Out of that we are just interested in current value hence the return type has been named accordingly and the doxygen for @return has that info.